docs/SUMMARY.md

```diff
@@ -4,8 +4,8 @@
 
 ## Docs
 
-* [Internal architecture](internal-architecture.md)
-  * [Index Tracker](index-tracker.md)
+* [Internal architecture](internal-architecture/README.md)
+  * [Index Tracker](internal-architecture/index-tracker.md)
```

Comment:
hhm, shoultn't this happen in the sync script that needs to fix these as they get broken when the docs get added to the main ones right?

Reply:
Looks like this is the default gitbook way of adding internal pages (you can see it by yourself when adding an internal page using gitbook dashboard). and also this will add complication to the sync process.

Reply:
so how come it works even without this extra folder before adding it to the main docs?
https://docs.tellor.io/docs-preview/v/docs-preview/

Reply:
I'm not sure why this works here and not there, maybe we should ask from the Gitbook discord.

Reply:
yep , please do and update the issue with the answer